com.unity.render-pipelines.universal/Runtime/UniversalRenderPipelineCore.cs

```diff
@@ -134,9 +155,10 @@ public bool IsCameraProjectionMatrixFlipped()
 
         public SortingCriteria defaultOpaqueSortFlags;
 
-        public bool isStereoEnabled;
-        internal int numberOfXRPasses;
-        internal bool isXRMultipass;
+        internal XRPass xr;
+
+        [Obsolete]
+        public bool isStereoEnabled { get => xr != null && xr.enabled; }
```

Comment:
Fix in this PR: this will cause an API validation fail.
For now keep the field (not property) as Obsolete. We can remove in 2021.1.

Reply:
Add a message to Obsolete attribute.